src/pages/StatusPage.tsx

```diff
@@ -0,0 +1,216 @@
+        github?.githubOcto?.issues?.listComments,
+        {
+          owner,
+          repo,
+          issue_number: testIssueNumber,
+        })
+
+      // getting last SSA and its time. 
+      const last_SSA = comments.reverse().find((issue: any) => issue.body.includes("SSA bot ran")).created_at
+      setLastSSAtime(last_SSA)
+      // check if it is less than 3 hours.
+      const is_Ssa_Healhty = moment.duration(moment(new Date()).diff(moment(last_SSA))).asHours() < 3.2
+      setIsSSAHealhty(is_Ssa_Healhty)
+
+    } catch (error) {
+      console.log(error)
+    } finally {
+      setIsSSALoading(false)
+    }
+  }
+
+  const checkHealthStatus = (isLoading: boolean, botStatus: boolean | null) => {
+    if (isLoading) {
+      return <div>Checking...</div>
+    } else {
+      if (botStatus) {
+        return <CheckCircleIcon sx={{ color: "green" }} />
+      } else {
+        return <DoNotDisturbOnIcon
+          sx={{ color: "red" }}
+        />
+      }
+    }
+  }
+
+  return (
+    <Box m="0 auto" marginTop="6rem" sx={{ minHeight: "30rem" }}>
+      <Typography variant="h4" mb="2rem" textAlign="center">
+        Service Status
+      </Typography>
+      <Box sx={{ width: "60rem" }}>
+        <Paper elevation={4}>
+          <Stack
+            sx={{ p: "1rem" }}
+            direction="row"
+            justifyContent="space-between"
+          >
+            <Typography variant="h6">
+              Current status by service
+            </Typography>
+            <Stack
+              direction="row"
+              alignItems="center"
+              spacing={4}
+            >
+              <Stack
+                direction="row"
+                alignItems="center"
+                spacing={1}
+              >
+                <CheckCircleIcon
+                  sx={{ color: "green" }}
+                />
+                <span>Active</span>
+              </Stack>
+              <Stack
+                direction="row"
+                alignItems="center"
+                spacing={1}
+              >
+                <DoNotDisturbOnIcon
+                  sx={{ color: "red" }}
+                />
+                <span>Down</span>
+              </Stack>
+            </Stack>
+          </Stack>
+          <Divider />
+          <Stack>
+            <Stack
+              sx={{ p: "1rem" }}
+              direction="row"
+              justifyContent="space-between"
+            >
+              <Typography variant="body1">
+                SSA BOT
+              </Typography>
+              {checkHealthStatus(isSSALoading, isSSAHealhty)}
+            </Stack>
+            <Divider />
+            <Stack
+              sx={{ p: "1rem" }}
+              direction="row"
+              justifyContent="space-between"
+            >
+              <Typography variant="body1">
+                LDN BOT
+              </Typography>
+              {checkHealthStatus(isLDNLoading, isLDNBotHealthy)}
```

Comment:
when we are waiting for the LDN bot to comment the issue, we should display a loadspinner